src/application/health_checks/health_checks.h

```diff
@@ -0,0 +1,28 @@
+#ifndef HEALTH_CHECKS_H
+#define HEALTH_CHECKS_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define ADC1_MAX_COUNTS 65535.0f
+#define ADC1_VREF 3.3f
+#define R_SENSE 0.033f
+#define INA180A3_GAIN 100.0f
+
+static inline float adc1_voltage_v(uint32_t counts) {
```

Comment:
I am confused about design intent with these inline functions and the ADC parameters - are they meant to be used outside the module?